js/script.js

```diff
@@ -0,0 +1,245 @@
+//New task added
+//Realized checked task property
+//Realized edit task posibility
+//Realized delete tasks posibility
+//If task is checked,  it cant'be removed
+//If task is checked, it remove to the end of list
+//Task can be edit. Realized posibility to escape task's change
+//All tasks edit event saved in storage
+
+let allTasks = JSON.parse(localStorage.getItem("tasks")) || [];
+let inputValue = "";
+let inputField = null;
+const addButton = document.querySelector(".add-button");
+
+const updateValue = (evt) => {
+  inputValue = evt.target.value;
+};
+
+const removeChild = (parentElement) => {
+  while (parentElement.firstChild) {
+    parentElement.removeChild(parentElement.firstChild);
+  }
+};
+
+const renderTask = () => {
+  const tasksList = document.querySelector(".tasks__list");
+
+  removeChild(tasksList);
+  allTasks.forEach((item, index) => {
+    const checkbox = document.createElement("input");
+    const imageEdit = document.createElement("img");
+    const imageDelete = document.createElement("img");
+    const imageEscape = document.createElement("img");
+    const imageDone = document.createElement("img");
+    const container = document.createElement("li");
+    const buttonEdit = document.createElement("button");
+    const buttonDelete = document.createElement("button");
+    const buttonEscape = document.createElement("button");
+    const buttonDone = document.createElement("button");
+    const textField = document.createElement("p");
+
+    container.className = "tasks__item";
+    container.id = `task-${index}`;
+
+    textField.id = `tasks__label-${index}`;
+    textField.innerText = item.text;
+    textField.className = item.isCheck
+      ? "text tasks__text tasks__text--done"
+      : "text tasks__text";
+    textField.classList.add("tasks-text");
+
+    checkbox.className = "tasks__checkbox";
+    checkbox.type = "checkbox";
+    checkbox.id = `tasks__checkbox-${index}`;
+    checkbox.checked = item.isCheck;
+    checkbox.onchange = () => onCheckboxChange(index);
+
+    imageEdit.className = "tasks__icon";
+    imageEdit.src = "img/edit.svg";
+    imageEdit.width = "15";
```

Comment:
in css file

Reply:
I can only image width in CSS. Because this element is not exist in HTML. Or I can set image as background, but I think that bad idea.